csunplugged/tests/utils/test_render_scratch_images.py

```diff
@@ -0,0 +1,91 @@
+"""Test class for render_scratch_images in check_required_files module."""
+
+from django.test import SimpleTestCase, override_settings
+from utils import check_required_files
+from unittest.mock import Mock
+import os.path
+import shutil
+
+SCRATCH_PATH = "temp/tests/scratch/"
+
+
+@override_settings(SCRATCH_GENERATION_LOCATION=SCRATCH_PATH)
+class RenderScratchImagesTest(SimpleTestCase):
+    """Test class for render_scratch_images in check_required_files module."""
+
+    def tearDown(self):
```

Comment:
Probably best to call `super().tearDown()` at the end of this function, in case the django base test classes do anything important with it.

Reply:
Nice spotting.